src/Helper/Uri.php

```diff
@@ -347,15 +372,22 @@ public function getHost(mixed $default = null): ?StringHelper
 
     /**
      * Returns TRUE if the port is valid.
-     * todo: Implement check for port URI segment
      *
      * @param integer|null $port
      * @return boolean
      */
     public function checkPort(int $port = null): bool
     {
         if ($port === null) {
-            $port = $this->port;
+            $port = $this->getPort();
```

Comment:
See comment as above regarding this use of `$this->getPort()` vs `$this->port`.

Reply:
Changed.